src/main/resources/static/assets/css/colors/blue-dark.css

```diff
@@ -304,4 +304,4 @@ a.link:hover, a.link:focus {
   position: fixed;
   font-size: 12px;
   left: 0;
-}
\ No newline at end of file
+}
```

Comment:
@snuyanzin we could leave the formatting on the vendor specific js, css, and any other files just in case to avoid any sort of behaviours in UI it might cause, unless there are vulnerabilities. 
On the other hand, as we are now working on new react based UI, we could leave these changes as is.